Makefile

```diff
@@ -145,8 +145,8 @@ migrate-down: ## migrate down using gomigrate.
 ##@ Codegen
 
 .PHONY: codegen
-codegen: gomock ## generate code using go generate (mocks).
-	go generate ./...
+codegen: mockgen ## generate code using go generate (mocks).
+	PATH=$(LOCALBIN_TOOLING):$$PATH go generate ./...
```

Comment:
what does this do?

Reply:
This is to ensure we can run `bin/tooling/codegen` - `bin/tooling` is not in my default PATH, so Make wasn't finding `codegen` (I did have an old `gomock` in my `$HOME/go/bin`, but we want to be sure we are using `bin/tooling/` for whatever tools we download/install in this Make processing.

Reply:
I think you can provide `GOBIN` to be equal to `LOCALBIN_TOOLING` and that should be considered first. (not 100% tho)

Reply:
`GOBIN=$(LOCALBIN_TOOLING) $(GO) generate ./...` but still get path error.